test/loader_test/test_image_objects.c

```diff
@@ -349,9 +349,10 @@ int test_image_objects()
         test_clEnqueueCopyBufferToImage(&clEnqueueCopyBufferToImageData[i]);
     }
 
-    for (i = 0; i<NUM_ITEMS_clEnqueueMapImage; i++) {
-        test_clEnqueueMapImage(&clEnqueueMapImageData[i]);
-    }
+    // Segfaults when freeing the returned pointer
```

Comment:
Again, could there be an issue filed about fixing whatever is wrong here?  Or maybe both the segfaults are one issue.